src/options.rs

```diff
@@ -51,10 +52,15 @@ impl GitOptions {
             Some(branch) => branch,
             None => "master".to_owned(),
         };
+        let ignored_branch = match opts.opt_str("i") {
```

Comment:
Yep, that's a much better way to do it. If you want to clean up the assignments around it, I wouldn't mind :)